index.html

```diff
@@ -1569,6 +1569,13 @@ <h2>Capabilities</h2>
   <td>Describes the <a>current session</a>’s <a>user prompt handler</a>.
    Defaults to the <a>dismiss and notify state</a>.
  </tr>
+
+ <tr>
+  <td><dfn>User Agent</dfn>
+  <td>"<code>userAgent</code>"
+  <td>string
+  <td>Identifies the <a>remote end</a>'s <a>default User-Agent value</a> of the <a>endpoint node</a>.
```

Comment:
An `endpoint node` is actually a subset of `remote end`. So I assume we can get rid of the latter similar to the other prefs?

```suggestion
  <td>Identifies the <a>default User-Agent value</a> of the <a>endpoint node</a>.
```